### PR TITLE
President return types

### PR DESCRIPTION
**Pull request by akowal3**
Same as #25 but merge to outgoing.
Outgoing synced with main.

### CODE REVIEW

File: internal/common/shared/iigopresidentreturn.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const (
+	PresidentAllocation PresidentReturnContentType = iota
+	PresidentTaxation
+	PresidentRuleProposal
+	PresidentSpeakerSalary
+)
+
+// PresidentReturnContent is a general datastructure used for president return type
+type PresidentReturnContent struct {
+	T             PresidentReturnContentType
```
Comment (Spruudis) [inline internal/common/shared/iigopresidentreturn.go]: What does T stand for?

File: internal/common/shared/iigopresidentreturn.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package shared
+
+// PresidentReturnContentType is the index of the return content that should be retrieved if the action was taken.
+type PresidentReturnContentType = int
+
+const (
```
Comment (Spruudis) [inline internal/common/shared/iigopresidentreturn.go:7]: Golint issue here

**Review by Spruudis — APPROVED** [review]
T should have a better name for readability but other than that and the golint documentation issue seems good to PR out

File: internal/common/shared/iigopresidentreturn.go (hunk 1 above)
Comment (akowal3) [reply internal/common/shared/iigopresidentreturn.go]: yeah. that was taken from the communications

File: internal/common/shared/iigopresidentreturn.go (hunk 2 above)
Comment (akowal3) [reply internal/common/shared/iigopresidentreturn.go:7]: fixed

File: internal/common/shared/iigopresidentreturn.go (hunk 1 above)
Comment (akowal3) [reply internal/common/shared/iigopresidentreturn.go]: changed to ContentType

**Pull request merged**